### DISCUSSION TITLE
Alpine js Wordpress enqueue 3xxx

### DISCUSSION THREAD

**Discussion by ahmedch1** [alpinejs/alpine#4065] [1. Help]
Hello,
I am getting this problem when analyzing my website 
3xxx javascript related to alpine js 
This is my code on wordpress in functions.php 

```
        add_action('wp_enqueue_scripts', function () {

            wp_register_script('alpine', '//unpkg.com/alpinejs', null, null, true);
            wp_enqueue_script('alpine');
        }, 100);
```

Does anyone has an idea how to resolve it without affecting the website functionnality ?
Thanks in advance

**Comment by SimoTod** [comment]
Use a different cdn, unpkg is flimsy.

You can use the one shown here ->
https://alpinejs.dev/start-here

**Comment by ekwoka** [comment]
Use ESM.sh.

Just import it directly or vendor the script into your WordPress instance.

**Comment by docdunning** [comment]
```
wp_register_script('alpine', '//unpkg.com/alpinejs', null, null, true);
wp_enqueue_script('alpine');

```
I've tried the same code in `functions.php` on one of my WordPress sites and I don't see any problem.

Where does `3xxx javascript` appear?

**Reply by SimoTod** [reply]
The OP didn't specify what the problem was but 3xx is just a redirect (using the url they posted, the cdn redirects to a different url before serving the file).
I suppose whatever tool they are using to analyse the site flags it as something to fix/optimise.
